src/queries/enum_repr_int_removed.ron

```diff
@@ -52,7 +62,8 @@ SemverQuery(
     }"#,
     arguments: {
         "public": "public",
-        "repr_regex": "#\\[repr\\([ui]\\d+|size\\)\\]",
+        "repr": "repr",
+        "repr_regex": "[ui]\\d+|size",
```

Comment:
Same regex concern as in the other file, we probably want tests for this as well.

Reply:
Did so, just like in the other file.